pom.xml

```diff
@@ -190,6 +191,7 @@
         <version.org.jboss.logging.jul-to-slf4j-stub>1.0.1.Final</version.org.jboss.logging.jul-to-slf4j-stub>
         <version.org.jboss.logmanager.jboss-logmanager>2.1.17.Final</version.org.jboss.logmanager.jboss-logmanager>
         <version.org.jboss.logmanager.log4j-jboss-logmanager>1.2.0.Final</version.org.jboss.logmanager.log4j-jboss-logmanager>
+        <version.org.jboss.logmanager.log4j2-jboss-logmanager>1.0.0.Beta3</version.org.jboss.logmanager.log4j2-jboss-logmanager>
```

Comment:
@jamezp, @boris-unckel  is it ok to use a "Beta" version for log4j2-jboss-logmanager?

Reply:
@tommaso-borgato nope, but I've upgraded it to final now :)